app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+class TweetController < ApplicationController
+
+  get "/tweets" do
+    if logged_in?
+      @user = current_user
+      erb :"/posts/index"
+    else
+      redirect to "/login"
+    end
+  end
+
+  get "/tweets/new" do
+    if logged_in?
+      erb :"/posts/new"
+    else
+      redirect to "/login"
+    end
+  end
+
+  post "/tweet" do
+    if params[:content] != ""
+      tweet = Tweet.new(content: params[:content])
+      tweet.user_id = current_user.id
+      tweet.save
+      redirect to "/tweets/#{tweet.id}"
+    else
+      redirect to "/tweets/new"
+    end
+  end
+
+  get "/tweets/:id" do
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :"/posts/show"
+    else
+      redirect "/login"
+    end
+  end
+
+  get "/tweets/:id/edit" do
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?